openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.utils.tsx

```diff
@@ -69,21 +68,3 @@ export const getTabs = (
 
   return [tabs.teams, tabs.users, ...commonTabs];
 };
-
-export const searchTeam = (teams: Team[], value: string): Team[] => {
-  let results: Team[] = [];
-  for (const team of teams) {
-    const hasChildren = !isUndefined(team.children) && !isEmpty(team.children);
-    const matched =
-      team?.name?.toLowerCase().includes(value.toLowerCase()) ||
-      team?.displayName?.toLowerCase().includes(value.toLowerCase());
-    if (matched) {
-      results = [...results, { ...team, children: undefined }];
-    }
-    if (hasChildren) {
-      results = [...results, ...searchTeam(team.children as Team[], value)];
-    }
-  }
-
-  return results;
-};
```

Comment:
Let's have this method around we need it in future

Reply:
sure!